index.js

```diff
@@ -67,6 +67,6 @@ module.exports = (logicDir, dataDir, opts) => {
     });
 
 
-    const port = 3000;
-    app.listen(port, () => console.log(`Server started`));
+    const port = opts.port ? opts.port : 3000;
```

Comment:
If I don't pass the port and If I'm running some other service on port 3000? We will get PORT already in use error.